net.gaijin.WarThunder.metainfo.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.gaijin.WarThunder</id>
+  <launchable type="desktop-id">net.gaijin.WarThunder.desktop</launchable>
+  <name>War Thunder</name>
+  <developer_name>Gaijin Entertainment</developer_name>
+  <summary>War Thunder is the most comprehensive free-to-play, cross-platform, MMO military game.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://warthunder.com</url>
+  <description>
+    <p>In War Thunder, aircraft, attack helicopters, ground forces and naval ships collaborate in realistic competitive battles. You can choose from over 1,900 vehicles and an extensive variety of combat situations many of which are exclusive. You can find yourself blasting your pursuers from a bomber turret, defending your teammates on the ground from an air raid with anti-aircraft guns, shooting down enemy planes with a firestorm from multiple rocket launchers, or trying to sink an enemy warship with a torpedo from a fast attack boat.</p>
+    <p>Features include:</p>
+    <ul>
+     <li>Cross-platform gameplay between Windows PC, Linux, Mac, PlayStation 4, Xbox One, PlayStation 5 and Xbox Series X|S users.</li>
+     <li>Over 2,000 highly detailed aircraft, helicopters, tanks, warships and other combat vehicles crafted carefully from historical documents and surviving sources.</li>
+     <li>Over 100 maps representing the main historical battle theaters.</li>
+     <li>Intense PvP experiences in full-scale combat missions at various difficulty settings for all play styles and degrees of experience.</li>
+     <li>Rich PvE content including dynamic historical campaigns and solo missions.</li>
+     <li>Regular content updates including new vehicles, maps, missions and nations.</li>
+     <li>Astonishing graphics, authentic sound effects and beautiful music creating an atmosphere to fully immerse yourself in.</li>
+     <li>Create custom content for War Thunder and share it on War Thunder Live, with the prospect of earning real money through the Revenue Share Partner System!</li>
+    </ul>
+   <p>NOTE: This package is not verified by, affiliated with, or supported by Gaijin.</p>
+   <p>NOTE: This installs the launcher into the user data directory. Expect storage usage of 14GB+</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://lh3.google.com/u/0/d/1eBTdcQuiconkJJnC8d768gxIcFl_ai6H=w1920-h1080-iv1</image>
```

Comment:
Would be nice to use some actual screenshots taken on Linux, but it's not a blocker.

Reply:
There isn't really much difference between Linux / Windows / MacOS of this game